app/src/main/res/values/strings.xml

```diff
@@ -605,4 +605,14 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_address_answer_no_housenumber_message1">The building was tagged as:</string>
     <string name="quest_address_answer_no_housenumber_message2">Is this correct and is it not just part of a building?</string>
 
+    <string name="quest_maxheight_title">"What is the height limit here?"</string>
+    <string name="quest_maxheight_parking_entrance_title">What is the height limit of this parking entrance?</string>
+    <string name="quest_maxheight_height_restrictor_title">What is the height limit of this height restrictor?</string>
+    <string name="quest_maxheight_tunnel_title">What is the height limit of this tunnel?</string>
+    <string name="quest_maxheight_answer_noSign">There is no sign</string>
+    <string name="quest_maxheight_answer_noSign_question">Can you give some more information?</string>
+    <string name="quest_maxheight_answer_restrictsTraffic">The physical height restricts the general traffic</string>
```

Comment:
So my thinking is: Why not ask the user to measure it? Or perhaps taking a photo would be posible (create a note).
I suggest the following:

> If you can instead measure the height, please do! Otherwise, is there enough space for even big trucks to pass?
> You can also leave a note and attach a photo to it.
> **Leave note | No | Yes**

And never tag any of `no_indications/unsigned/none/no_sign`, because that information is pretty worthless.

Reply:
Should there be a new dialog for measuring the height? And the tag for that purpose would be something like `maxheight:estimated`? Or should we still use `maxheight`?

Reply:
I would use an extra tag like `maxheight:estimated` because [`maxheight`](https://wiki.openstreetmap.org/wiki/Key:maxheight) is...

> The **legal** maximum height

...and nothing estimated. But maybe we would have to create a new wiki page for this tag because according to [taginfo](https://taginfo.openstreetmap.org/keys/maxheight%3Aestimated) this tag is currently used 0 times 😄
Maybe we could just change/add a hint like it is done for e.g. the vegetarian quest if the user wants to estimate the height but keep the layout as it is...

Reply:
Though, an estimated height is, beyond stating that "it is high enough that anything can pass" next to useless. Basically, an estimated height can and should only be used as a marker for other people to supply the proper height.
So, perhaps it is the best to let the user just make the estimation whether it is **easily** (adapt wording) high enough for any vehicle to pass through or not. Only if the user had a measurement tape, it could be possible for him to add the precise height (and indeed that would be valuable information).
So, if the user answered no, mark it as, dunno, `no_sign` or something and this tag along with the others (`no_indications, unsigned, ...`) can be used as a marker for other surveyors that **do** bring a measurement tape to solve it. Maybe this could be a second "advanced" (default disabled) quest.